init/keycloak/kc-seed.sh

```diff
@@ -7,5 +7,26 @@ do
 done
 cd /opt/jboss/keycloak/bin
 ./kcadm.sh config credentials --user admin --password admin --server http://localhost:8080/auth --realm master
-./kcadm.sh create clients -s clientId=invitease -s 'redirectUris=["http://invitease.localhost/*"]' -s clientAuthenticatorType=client-secret -s serviceAccountsEnabled=true -s secret=d0b8122f-8dfb-46b7-b68a-f5cc4e25d000
+USER_ID=$(./kcadm.sh get users -r master --limit 1 --fields id | grep -oP '(?<="id" : ")[^"]*')
+./kcadm.sh create clients \
+    -s clientId=invitease \
+    -s 'redirectUris=["http://invitease.localhost/*"]' \
+    -s clientAuthenticatorType=client-secret -s serviceAccountsEnabled=true \
+    -s secret=d0b8122f-8dfb-46b7-b68a-f5cc4e25d000 
+CLIENT_ID=$(./kcadm.sh get clients -r master -q clientId=invitease --fields id | grep -oP '(?<="id" : ")[^"]*')
+./kcadm.sh create groups -r master -s name=admin
+GROUP_ID=$(./kcadm.sh get groups -r master -q name=admin | grep -oP '(?<="id" : ")[^"]*')
+./kcadm.sh update users/$USER_ID/groups/$GROUP_ID -r master -s realm=master -s userId=$USER_ID -s groupId=$GROUP_ID -n
+./kcadm.sh create clients/$CLIENT_ID/protocol-mappers/models -r master \
```

Comment:
What do you think about adding phone, user_role (admin, guard, user) and NPP/NPR field already as part of this PR?

Reply:
I'm not sure what you mean

Reply:
Not sure what you mean

Reply:
Here we're defining the user fields, am I correct?

Reply:
No, I just create a group and add the admin user to it